docs/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Changelog
 
-## v1.3
-## 27/09/2020
+## [v1.3](https://github.com/zd-zero/waifu-motivator-plugin/releases/tag/v1.3)
+#### 27/09/2020
 
-Features:
+:sparkles: Features:
```

Comment:
This is an awesome feature of GH where you can use the emojis between `:` character, but I suggest to use the ✨ and 🐛 characters instead, so we can still see the emojis even outside the GH platform.